activejob/CHANGELOG.md

```diff
@@ -205,4 +205,10 @@
 
     *Jacopo Beschi*
 
+*   Clarify the backoff strategy for the recommended `:wait` option when retrying jobs
```

Comment:
This should be in the top of the file.

Reply:
Done in https://github.com/rails/rails/pull/49292/commits/a555ecb6a505ee92968d9daad44365c918ac0c5d